### DISCUSSION TITLE
What about "html frameworks" (htmx)

### DISCUSSION THREAD

**Discussion by AnthonyLenglet** [Devographics/surveys#174] [State of HTML Design]
Source: https://htmx.org/

Example from the quickstart:
```html
<button hx-post="/clicked" hx-swap="outerHTML">
  Click Me
</button>
```

I was about to fill this in the features category, but I don't actually know how to deal with this case
SoJS has a frameworks category, and so does SoCSS, based on this would it make sense to add htmx in SoHTML ?

I don't actually know if there is a ton of frameworks like this so it might seem weird if there is only one, hence the reason why I'm not sure how to handle this

**Comment by LeaVerou** [comment]
[Mavo](https://mavo.io) is another one, and precedes HTMx (to my knowledge). There's also [AlpineJS](https://alpinejs.dev/) and a few more. Right now I have these under [a more general question about producing HTML](https://github.com/Devographics/surveys/discussions/165) but we may want to break it down.

**Comment by LeaVerou** [comment]
Started prototyping questions around this here: https://coda.io/@leaverou/state-of-html/tools-4
